src/components/ColocGWASTable.js

```diff
@@ -96,7 +96,7 @@ const getDownloadData = data => {
 const ColocGWASTable = ({ loading, error, fileStem, data }) => {
   const downloadData = getDownloadData(data);
   return (
-    <>
+    <React.Fragment>
```

Comment:
Not sure why this was changed. `<></>` is valid syntax for Fragments https://reactjs.org/docs/react-api.html#reactfragment